package.json

```diff
@@ -8,7 +8,7 @@
     },
     "scripts": {
         "start": "USE_PROTOTYPE=true NODE_ENV=production ./node_modules/.bin/ts-node -r tsconfig-paths/register src/main/server.ts",
-        "start:dev": "REDIS_LOCAL=true && nodemon --config nodemon.json",
+        "start:dev": "REDIS_LOCAL=true SESSION_COOKIE_SAME_SITE= && nodemon --config nodemon.json",
         "start:dev-windows": "set REDIS_LOCAL=true && nodemon --config nodemon.json",
```

Comment:
Not sure if we still want to keep "start:dev-windows". If we do might want to add the new setting to this one too

Reply:
Good spot. I've removed, as it's not used and not possible to test